CMakeLists.txt

```diff
@@ -59,7 +60,7 @@ PROJECT(${PROJECT_NAME} ${PROJECT_ARGS})
 APPLY_DEFAULT_APPLE_CONFIGURATION()
 
 # Add the different required and optional dependencies
-ADD_PROJECT_DEPENDENCY(pinocchio 2.4.3 REQUIRED PKG_CONFIG_REQUIRES "pinocchio >= 2.4.3")
+ADD_PROJECT_DEPENDENCY(pinocchio 2.4.2 REQUIRED PKG_CONFIG_REQUIRES "pinocchio >= 2.4.2")
```

Comment:
Please revert back this needed version.
We define the minimum required version.
You need to update your pinocchio version :)